adyenv6core/src/com/adyen/v6/repository/NotificationItemRepository.java

```diff
@@ -56,30 +56,30 @@
         return nonProcessedNotifications;
     }
 
-    public  boolean isNewerNotificationExists(String merchantReference, Date eventDate, String merchantAccountCode)
-    {
-        final Map queryParams = new HashMap();
-        queryParams.put("merchantReference", merchantReference);
+    public boolean isNewerNotificationExists(String merchantReference, Date eventDate, String merchantAccountCode) {
```

Comment:
This is generic method for all types of events. Should we also add specific check for report available notifications? otherwise it will also pass other types.

Reply:
Would that be wrong to pass for other cases as well? 
It shouldn't happen anyway, unless something is changed in the future, right now the other notifications always come with merchantReference.